src/lazy/encoding.rs

```diff
@@ -150,6 +154,14 @@ impl Encoding for BinaryEncoding_1_0 {
     fn default_write_config() -> WriteConfig<Self> {
         WriteConfig::<Self>::new()
     }
+
+    fn default_value_writer_config() -> ValueWriterConfig {
```

Comment:
🗺️ Each encoding can now provide its default `ValueWriterConfig`, which is what a `Writer` starts with.